magento/magento2
# issue #25003 fixed

Wrong currency symbol in creditmemo_grid & sales_order_view > creditmemo grid for Adjustment Refund & Adjustment Fee columns
### Description (*)

For Columns Adjustment Refund & Adjustment Fee in creditmemo_grid & sales_order_view > credit memo grid if the order is placed with the different currency.
### Fixed Issues (if relevant)

1. Fixes #25003 : Wrong currency symbol in creditmemo_grid & sales_order_view > credit memo grid for Adjustment Refund & Adjustment Fee Columns

### Manual testing scenarios (*)

1. Set USD as Base Currency
2. Set INR currency for different store
3. Place an order from INR Currency store.
4. Go to Backend > SALES > Credit Memos, create Credit Memo with Adjustment Refund & Adjustment Fee
5. Check Adjustment Refund & Adjustment Fee column on Credit Memo grid (sales_creditmemo_index) & sales_order_view > credit memo grid.

### Questions or comments

Finally, Adjustment Refund & Adjustment Fee columns value is showing correctly currency symbol.

### Contribution checklist (*)
